test/functional/test_framework/mininode.py

```diff
@@ -412,3 +419,24 @@ def run(self):
             [obj.handle_close() for obj in disconnected]
             asyncore.loop(0.1, use_poll=True, map=mininode_socket_map, count=1)
         logger.debug("Network thread closing")
+
+def network_thread_start():
+    """Start the network thread."""
+    # Only one network thread may run at a time
+    assert not network_thread_running()
+
+    NetworkThread().start()
```

Comment:
Why not keep this instance and check for it, instead of enumerating threads with a custom name?

Reply:
I can't see how that's any better. Perhaps you can push a branch to demonstrate what you mean.